Format.pm

```diff
@@ -505,24 +536,50 @@ variables, use C<format_number()> instead.
 
 sub round
 {
-    my ($self, $number, $precision) = _get_self @_;
+    my ($self, $number, $precision, $roundoption) = _get_self @_;
 
     unless (defined($number))
     {
         _complain_undef();
         $number = 0;
     }
 
-    $precision = $self->{decimal_digits} unless defined $precision;
-    $precision = 2 unless defined $precision;
+    $precision   = $self->{decimal_digits} unless defined $precision;
+    $precision   = 2 unless defined $precision;
+    $roundoption = $self->{round_option} unless defined $roundoption;
+    $roundoption = ROUND_NORMAL unless defined $roundoption;
 
     croak("precision must be integer")
         unless int($precision) == $precision;
 
     if (ref($number) && $number->isa("Math::BigFloat"))
     {
         my $rounded = $number->copy();
-        $rounded->precision(-$precision);
+
```

Comment:
Me to, but calling a method without using the result also produce strong feelings :)

And i think maintainable code is more important than writing 10 more lines of code to do the correct thing to avoid a possible second function call (in a really rare case).